src/misc_cmd.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * This file is part of OpenTTD.
+ * OpenTTD is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 2.
+ * OpenTTD is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with OpenTTD. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file misc_cmd.h Miscellaneous command definitions. */
+
+#ifndef MISC_CMD_H
+#define MISC_CMD_H
+
+#include "command_type.h"
+#include "economy_type.h"
+
+enum PauseMode : byte;
+
+enum class LoanCommand : byte {
+	Interval,
```

Comment:
Isn't coding style that all enums are all uppercase?

Reply:
This is an `enum class` so the enum declaration forms a namespace. You can only refer to the members qualified, such as `LoanCommand::Interval`. In my opinion the convention of uppercase for constants falls away in that case, since it's no longer (as) necessary to make the enum members unique.

Reply:
Current precedent in the code is that `enum class` uses normal casing for the enum members.

Reply:
That precedent is inconsistent though. Both "normal" case and upper case exist for `enum class`. So changing the current upper case ones to "normal" case and documenting it in the coding style could be a solution to my remark, though for a separate PR.

Reply:
`TCPConnecter::Status` and  `IndustryDirectoryWindow::SorterType` use upper case, everything else uses normal cases as far as I can see.